core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -142,8 +156,11 @@ class LoadBalancerService(config: WhiskConfig, verbosity: LogLevel)(
      * A map storing current activations based on ActivationId.
      * The promise value represents the obligation of writing the answer back.
      */
-    case class ActivationEntry(created: Instant, promise: Promise[WhiskActivation])
-    private val activationMap = new TrieMap[ActivationId, ActivationEntry]
+    case class ActivationEntry(id: ActivationId, subject: String, invokerIndex: Int, created: Instant, promise: Promise[WhiskActivation])
+    type ActivationSet = TrieMap[ActivationEntry, Unit]
```

Comment:
As this is pretending to be a `Set`, would it help readability of we implement a tiny stub around this?

Something like:

```
class TrieSet[A] {
	private val backing = new TrieMap[A, Unit]
	
	def add(elem: A) = backing.put(elem, ())
	def remove(elem: A) = backing.remove(elem)
}
```

Even if we don't do that, I'd propose to change this line to:

`type TrieSet[A] = TrieMap[A, Unit]`

Glancing over the code, when reading `ActivationSet` my first thought was that this is a `case class`.